caas/kubernetes/provider/modeloperator.go

```diff
@@ -523,18 +536,23 @@ func modelOperatorService(
 	}
 }
 
+func modelOperatorGlobalScopedName(model, operatorName string) string {
+	return fmt.Sprintf("%s-%s", model, operatorName)
```

Comment:
this will be a bit meh if model is ""
(which the code expects cause it does a model nil check)
so let's add a small tweak here